### ISSUE TITLE
ExpressJS crashes when using libxslt along with libxmljs

### ISSUE THREAD

**Issue by benoror** [albanm/node-libxslt#4]
Hello Alban!

Yesterday I was stuck with my app crashing for _apparently no reason_. I'm writting an API with ExpressJS. 

It suddenly crashes (_no err message, no stacktrace_) when calling **`libxslt.parse`** or **`libxslt.parseFile`**

After hours of digging in I found that removing **`var libxmls = require('libxmljs')`** solved the issue completely!

(The reason I required **libxmljs** is that a need to parse XML _somewhere else in my code_. According to [this example](https://github.com/albanm/node-libxslt#libxmljs-integration) I supposed that would be possible.)

Is there some **conflict by using libxmljs along with libxslt** that I should be aware of?

PS: Great work BTW!

Thanks in advance, have a great day!

**Comment by albanm** [comment]
Hello,

Oops, I would at least have expected a trace with a segfault or something like that, this is weird.

Anyway I just released a 0.4.0 version that exposes its internal libxmljs required module.
Maybe this will work as a workaround for you ? To test it you simply need to do something like this:

    var libxslt = require('libxslt');
    var libxmljs = libxslt.libxmljs;

Please tell me if this solves your problem. I will try to have a closer look soon.

Sorry for the inconvenience.

Have a great day too !

**Comment by benoror** [comment]
Hi Alban,

Thanks for your response.

Goog news! **The issue got solved when I reinstalled `libxslt`** (thus rebuilding `libxmljs` via `node-gyp`)

Maybe it's worth mentioning that I use [nvm-windows](https://github.com/coreybutler/nvm-windows) to install and use the 32bit build of Node, since I never was able to compile `libxmljs` on 64bit.

Anyway, I just updated to 0.4.0 and I'm now using `libxslt.libxmljs` instead and works flawlessly.

Thanks again and kudos!
